Mathlib/Order/BooleanSubalgebra.lean

```diff
@@ -363,8 +363,8 @@ lemma closure_bot_sup_induction {p : ∀ g ∈ closure s, Prop} (mem : ∀ x hx,
 
 section sdiff_sup
 
-variable (sup : SupClosed s) (inf : InfClosed s) (bot_mem : ⊥ ∈ s) (top_mem : ⊤ ∈ s)
-include sup inf bot_mem top_mem
+variable (isSublattice : IsSublattice s) (bot_mem : ⊥ ∈ s) (top_mem : ⊤ ∈ s)
```

Comment:
I suppose outside of this there isn't much use for an `IsBoundedSublattice` predicate, which would also bundle the `bot_mem` and `top_mem`?